eng/pipelines/common/xplat-setup.yml

```diff
@@ -170,7 +170,7 @@ jobs:
 
         # OSX Public Build Pool (we don't have on-prem OSX BuildPool).
         ${{ if and(in(parameters.osGroup, 'osx', 'maccatalyst', 'ios', 'iossimulator', 'tvos', 'tvossimulator'), eq(variables['System.TeamProject'], 'public')) }}:
-          vmImage: 'macos-12'
+          vmImage: 'macos-13'
```

Comment:
I think we'll need to bump the official build image too since I see the same deprecation warning in the official build

Reply:
I'll run a test official build for this branch and the 8.0 one to make sure nothing breaks.

Reply:
Official builds look good. Merging